src/test/regress/expected/ensure_no_intermediate_data_leak.out

```diff
@@ -28,3 +28,11 @@ $$) WHERE result <> '';
 ---------------------------------------------------------------------
 (0 rows)
 
+-- ensure that we didn't leak any schemas in repartition joins
```

Comment:
shouldn't we run this query on the workers?

Reply:
Right